### ISSUE TITLE
Auto Heat Map Mode

### ISSUE THREAD

**Issue by charlesdaniels** [charlesdaniels/textvis#15]
Depends on #13. 

Once heat mapping is implemented, it would be useful to allow overriding the provided values with automatically computed ones. For example, link count. This would be configurable as a checkbox or something to that effect.

**Comment by charlesdaniels** [comment]
I'm going to assign this to @LothropRO for now, since you have your fingers in the heatmap stuff.

To implement this, I would implement a div that scrolls with the page view, and can be toggled by pressing a button (maybe a hamburger menu or something like that). Inside of that div, you can create a checkbox for this flag. You can create a separate issue for this if you would like. Look at the implementation of SVG annotations for a specific example of how to do this (you'll need to tweak this to be pegged to the screen as it scrolls). Remember that the menu button also needs to scroll with the screen -- I would peg a div to the bottom right corner of the screen with this button in it. This will also be useful for #16, since it's config options can be added to the same div.

Remember that the callers passes us a div they want us to do our thing into. You should make your new div a peer of the SVG annotation and tooltip, and a child of the one the caller gives us.

You should have a callback on the checkbox, so that when it is unchecked it runs through the data and sets the background colors as described in #13, and when it is checked it runs through the data and sets the background colors based on the link count over the whole paragraph, run through a mapping to a 0..1 scale.

You will need to know the max and min link count across the entire document to produce the mapping. You might compute that while the data is being initially loaded, or in another separate pass. You can certainly annotate any of the data structures or create new ones for convenience or performance at your discretion.

Note that this should update live, the user should not need to re-upload the document.
